Origins as a Subprotocol SIP (#35)
DistributedCollective/SIPS: SIP-0035.md

## The advantage for SOV token holders &amp; stakers

- Origins is part of the Sovryn protocol. And will always be linked to it through a Bonding Curve, thus having everlasting economic incentive. SOV is the coordination token of the Sovryn community, and as more projects are launched, the use of SOV to coordinate them will grow. 
- Veto Governance power over the Origins Governance.
- Staking Reward from Origins Revenue
- The fee collected on sell transactions in the Bonding Curve. 
- Opportunity for allocations from token launches on the Origins platform.

## Why implement a Bonding Curve?

The bonding curve inspiration is taken from the [forum post](https://forum.sovryn.app/t/new-utility-for-sov-minting-subprotocol-tokens/1611) created by John Light, Sovryn Contributor. To summarize the benefits:

- The bonding curve creates an economic link between SOV and the Origins subprotocol.
- This encourages the SOV holders who support the subprotocol development, at the same time isolates the subprotocol incentives.
- Generates demand for SOV, providing additional utility for the token.

## How is this bullish for SOV?

> Origins will create a new way to use SOV as a coordination token to develop the Sovryn protocol further, provide more governance capability and introduce new projects that grow the system. When the Origins platform launches subprotocols, they will frequently have their own governance token turning bitocracy to governance platform and creating new ways to use SOV. All this will allow the Sovryn protocol to scale, and at the same time, reduce the need for all decisions to pass through the SOV Bitocracy - which will become the highest layer of governance.

Reviewer: ```suggestion
Origins will create a new way to use SOV as a coordination token to develop the Sovryn protocol further, provide more governance capability and introduce new projects that grow the system. When the Origins platform launches subprotocols, they will frequently have their own governance token, turning Bitocracy into a governance platform and creating new ways to use SOV. All this will allow the Sovryn protocol to scale, and at the same time, reduce the need for all decisions to pass through the SOV Bitocracy - which will become the highest layer of governance.
```

> ## Sovryn Sub Protocol Launch on Origins

Reviewer: ```suggestion
## Sovryn Subprotocol Launch on Origins
```

> There will be multiple sub-protocols from Sovryn coming in the future. And as an added advantage, all the sub-protocols of Sovryn will not pay any mandatory fee for listing in Origins.

Reviewer: ```suggestion
There will be multiple subprotocols from Sovryn coming in the future. And as an added advantage, all the subprotocols of Sovryn will not pay any fee for listing in Origins.
```

Took out "mandatory",  since if it isn't required, it isn't mandatory.

## Process for Project Owners

> These are the Steps ideally a project should follow:

Reviewer: ```suggestion
These are the steps a project should ideally follow:
```

> 1. The project wishing to list on Origins should contact the Origins Team for guidance and to get any help they require to write an OIP which could have a good chance to get accepted by the Community. This is an optional step, but one highly recommended.

Reviewer: ```suggestion
1. The project wishing to list on Origins should contact the Origins Team for guidance and to get any help they require to write an OIP which could have a good chance to get accepted by the Community. This is an optional step, but one highly recommended.
```

Idea: Call these "OGP" for "Origins Governance Proposal" instead of OIP "Origins Improvement Proposal". Goes with the project branding since the token is called OG.

> 2. The concerned team then needs to make a post in the Forum, with the relevant draft OIP, Tokenomics, and any other relevant data.

Reviewer: Will Origins have its own forum? Or will it be a subcategory in the Sovryn forum?

Reviewer: ```suggestion
2. The concerned team then needs to make a post in the forum, with the relevant draft OIP, tokenomics, and any other relevant data.
```

3. The community will provide constructive criticism and present their views. It is recommended to be highly active in the forum at this time.
4. After a certain period, the Project Team can create a proposal in the Origins Governance and let the OG Stakers decide.
5. If the vote is positive, the project gets listed.

## Proposed Initial Origins Revenue Distribution

> The revenue earned from origins is proposed to be divided as below:

Reviewer: ```suggestion
The revenue earned from Origins is proposed to be divided as below:
```

- 20% of the revenue goes to SOV Stakers through Staking Reward

> - 10% of the revenue goes to buy OG from the Bonding Curve and then burning bought OG Tokens to increase the reserve and make OG Deflationary.

Reviewer: Why buy OG from the curve instead of the secondary market e.g. Sovryn AMM?

Reviewer: ```suggestion
- 10% of the revenue goes to buy OG from the Bonding Curve and then burning bought OG tokens to increase the reserve and make OG deflationary.
```

> - 50% of the revenue goes to OG Stakers through Staking Reward (including Vesting)

Reviewer: ```suggestion
- 50% of the revenue goes to OG Stakers through Staking Reward (including Vesting Stakers)
```

Reviewer: ```suggestion
- 50% of the revenue goes to OG Stakers through Staking Reward (including Vesting Stakers)
```

> - 50% of the revenue goes to OG Stakers through Staking Reward (including Vesting)

Reviewer: ```suggestion
- 50% of the revenue goes to OG Stakers through Staking Reward (including Vesting Stakers)
```

> - 50% of the revenue goes to OG Stakers through Staking Reward (including Vesting)

Reviewer: What does "including Vesting" here mean?

- 20% of the revenue goes to OG Treasury

## Tokenomics

> You can find the detailed version of this Tokenomics [here](https://docs.google.com/spreadsheets/d/1N3XcUFnAWSCPzUIeImZREAifrn30Vo2xN3XHu70ToxY/).

Reviewer: ```suggestion
You can find the detailed version of this tokenomics [here](https://docs.google.com/spreadsheets/d/1N3XcUFnAWSCPzUIeImZREAifrn30Vo2xN3XHu70ToxY/).
```

Reviewer: Do you expect the tokenomics to change? If not, it would be better to put the precise details directly into the SIP, in case the spreadsheet is changed or disappears for some reason.

Reviewer: If the tokenomics can change, you should say so in the SIP.

All the tokens have their vesting/locked schedule with unlocking & receiver party details.

## Token Sale

> Token Sale will happen in different rounds, with varying amounts of tokens, investment limits. For Ex: There will be a safe listing process for Sovryn Stakers, which is yet to be defined.

Reviewer: ```suggestion
Token Sale will happen in different rounds, with varying amounts of tokens, investment limits. For Ex: There will be a safe listing process for SOV Stakers, which is yet to be defined.
```

### Early Funders

The Early Funders token allocation (approximately 20%) is divided thusly:

| Name | Party | Deposit | Token Amount | Price | Invest Limit | Total Raised | Vesting Period (Years) | Vesting Cliff (Months) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Seed | Influential Investor &amp; Sovryn Team | Stable Coin | 15 Million | $0.2 | Min $50K | $3 Million | 2 | 4 |
| Presale | Sovryn Stakers | SOV | 11 Million | $0.22 | Max $5K | $2.42 Million | 1.5 | 2 |
|||||| **Total** | $5.42 Million |
|

> This will, in total, raise 5.42 Million.

Reviewer: ```suggestion
This will, in total, raise $5.42 Million (USD).
```

> The SOV raised in the Early Funders round will be primarily used for the  Origins Bonding Curve.

Reviewer: ```suggestion
The SOV raised in the Early Funders round will be primarily used for the Origins Bonding Curve reserves.
```

> The Stablecoin raised in the Early Funders round will be used by the  Origins Treasury.

Reviewer: ```suggestion
The stablecoins raised in the Early Funders round will be used by the Origins Treasury.
```

### Programmatic Sale

The Programmatic Sale token allocation is this way:

| Deposit | Token Amount | Price | Invest Limit | Total Raised | Vesting Period (Year) | Vesting Cliff (Months) |
| --- | --- | --- | --- | --- | --- | --- |
| SOV | 17 Million | $0.3 | $10K | $5.1 Million | 1 | 1 |
|||| **Total** | $5.1 Million |
|

> The SOV raised by Programmatic Sale will be primarily used for the  Origins Bonding Curve.

Reviewer: ```suggestion
The SOV raised by Programmatic Sale will be primarily used for the Origins Bonding Curve reserve.
```

## Budget

The anticipated budget for one year is around $2 Million, and once TGE occurs, it will require an OIP (Origins Improvement Proposal) through the Origins Bitocracy.

## Proposal

In general, the mission for this SIP is to get the below things:

- Approve the Origins as a subprotocol.

> - Ask the Sovryn Treasury (Exchequer) for 220K SOV in exchange for 40% of the OG Tokens solely used in the Origins <> SOV Bonding Curve.

Reviewer: ```suggestion
- Ask the Sovryn Treasury (Exchequer) for 220K SOV in exchange for 40% of the OG Tokens, solely to be used as reserves in the OG <> SOV Bonding Curve.
```
